src/libraries/Native/AnyOS/System.Globalization.Native/pal_calendarData.h

```diff
@@ -70,25 +70,25 @@ typedef enum
 // the function pointer definition for the callback used in EnumCalendarInfo
 typedef void (*EnumCalendarInfoCallback)(const UChar*, const void*);
 
-PALEXPORT int32_t GlobalizationNative_GetCalendars(const UChar* localeName,
+EXTERN_C PALEXPORT int32_t GlobalizationNative_GetCalendars(const UChar* localeName,
```

Comment:
Why do we need `EXTERN_C` ? Should rather be part of PALEXPORT ?

Reply:
We need extern “C” for Windows. When compiling VC++ since these files are .c files, it treats them as C. So when trying to link to coreclr the linker would complain it can’t find the exported types.

I didn’t put it as part of PALEXPORT as we override it to be empty when building S.G.Native from coreclr.

Reply:
So can we override `PALEXPORT` to `EXTERN_C` when building from coreclr? The point of PALEXPORT is that we can always set it to whatever makes the compiler happy, without touching all places.

Reply:
Ok sounds like a good idea. I’ll try that, thanks!